components/src/core/components/Topbar/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$oxd-topbar-bg-color: #ff6500;
+$oxd-topbar-bg-gradient: linear-gradient(90deg, #ff8226 0%, #f35c17 90%);
+$oxd-topbar-border-color: #e9eaec;
+$oxd-topbar-patch-color: #f7802a;
+
+$oxd-topbar-tab-text-color: #7a7b88;
```

Comment:
is these colors out of https://oxd-prod-infinity.orangehrm.com/pages/colors.html?

Reply:
These colours were not in design guide. but was used in the mock up design.

Reply:
We may have to think about how we make theses colors when it's a different theme